app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    lover_list = ["husband", "wife"]
+    for data in people:
+        person = Person(data["name"], data["age"])
+        for elem in lover_list:
+            if elem in data and data[elem] is not None:
```

Comment:
You could use walrus operator to make code more readable. Check it out here: [](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions)

`            if name:= data.get(elem):
                if elem == "wife" and name in Person.people:
                    person.wife = Person.people[name]
                    Person.people[name].husband = person`